unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -73,6 +73,39 @@ export function ComboboxValuesTest() {
       });
     });
 
+    describe('when calling clearAll without values', () => {
+      function addChildToCombobox() {
+        comboboxValues.element.appendChild($$('div').el);
+      }
+
+      beforeEach(() => {
+        initializeComponent(['updateAccessibilityAttributes']);
+        addChildToCombobox();
+        comboboxValues.clearValues();
+      });
+
+      it('should log analytics', () => {
```

Comment:
Im thinking it's a copy paste error ? The test name does not fit.

Reply:
I didn't even copy and paste this, my mind must've been somewhere else because I even vaguely remembered as if this was testing some analytics even though it obviously wasn't. Fixing it.